src/zon.zig

```diff
@@ -6,6 +6,7 @@ const NeverFailingAllocator = main.heap.NeverFailingAllocator;
 const List = main.List;
 
 pub const ZonElement = union(enum) { // MARK: Zon
+	uint: u64,
```

Comment:
I strongly discourage specializing integer types for Zon. That sickness is already a problem with string and stringOwned, where to properly handle strings you need to ensure both branches are handled and there is no warning that you are not handling one of them.
If you do that make sure to check every switch on zon variables anywhere to ensure you are not suddenly missing a special case. Considering that we don't have unit tests, it may come up in the future anyway.

Reply:
This seemed to be the most correct solution to me.
The zon parser right now can only handle i64. We either need to limit the range of values we allow to be processed, or we need to allow for unsigned integers in the zon parser.

I saw no other way to return a u64 than to add it to ZonElement.

Reply:
> If you have a sign, parse the string as i64, otherwise parse it as u64

I tried this before but it broke biome placement.